.pre-commit-config.yaml

```diff
@@ -48,7 +56,7 @@ repos:
     hooks:
       - id: clj-kondo-docker
 
-  - repo: https://github.com/allie-jo/cljfmt-docker
+  - repo: https://github.com/nikromen/cljfmt-docker
```

Comment:
Is this still needed since we found out that there actually was an error in the code and the image probably worked correctly?

Reply:
I applied fix from @praiskup there https://github.com/allie-jo/cljfmt-docker/pull/5